incubator/java-openliberty/templates/default/src/test/java/it/dev/appsody/starter/EndpointTest.java

```diff
@@ -0,0 +1,84 @@
+package it.dev.appsody.starter;
```

Comment:
Should we use this as an opportunity to add copyrights?

Reply:
Of course, this isn't new in 2020.. do we put the date that we first added the copyright header or the date in which the code originated (plus the date modified)?

Reply:
I dont have a strong opinion one way or another, but what would be the reason for adding copyrighting? Since this is part of a template project, would we necessarily need to add that? And does adding a copyright mean anything to an end user who is using/modifying the template?

Reply:
Don't worry for now.  Opened [issue](https://github.com/appsody/stacks/issues/633) to discuss more.